lib/segment/src/types.rs

```diff
@@ -451,6 +451,7 @@ pub struct VectorDataInfo {
 #[derive(Debug, Serialize, JsonSchema, Anonymize, Clone, PartialEq, Eq)]
 #[serde(rename_all = "snake_case")]
 pub struct SegmentInfo {
+    pub segment_uuid: String,
```

Comment:
I suggest to drop the `segment_` prefix.

```suggestion
    pub uuid: String,
```

Perhaps in other places too.